Aws ec2 right sizing (#246)
metricly/metricly-docs: content/right-sizing/reports-ec2-right-sizing.md

- **Name**: A text-input that selects resources with that match any text provided, such as a prefix (e.g., `west-app`).
- **EC2s**: A dropdown of all EC2s for manual selection with the option to select all.
- **Attributes**: A dropdown of all attributes for manual selection with the option to select all.
- **Tags**: A dropdown of all tags for manual selection with the option to select all.

### 2. CPU and Memory Constraints

The following constraint options are available for both CPU and Memory statements.

- **No Constraint**: Returns all possible recommendations for the given resource.
- **Cannot Increase CPU/Memory**: Returns recommendations where maximum resource utilization is at or below the input amount.
- **Minimum CPU/Memory**: Returns recommendations where minimum resource utilization is at or above the input amount.
- **Resize based on historical utilization**: Returns recommendations based on historical utilization of the given resource type.

#### Resource Specific Constraints

- **Target CPU Utilization**: Returns recommendations that closely match the target CPU utilization percentage.

> - **Default memory utilization when diagnostic monitoring not enabled**: Returns recommendations based on an assumption (as a percentage) of memory utilization.

Reviewer: For AWS, the CloudWatch agent is what provides memory as opposed to diagnostics in Azure:
```suggestion
- **Default memory utilization when the CloudWatch agent is not installed**: Returns recommendations based on an assumption (as a percentage) of memory utilization.
```
